### PR TITLE
Examples for vcloud plugin ver.1.3.

### PR DESCRIPTION
**Pull request by kostya13**
Update examples to latest version 1.3m6

### CODE REVIEW

File: types/vcloud-types.yaml [hunk 1]
```diff
@@ -20,11 +19,8 @@ node_types:
           hardware:
             cpu: { get_input: server_cpu }
             memory: { get_input: server_memory }
-          guest_customization:
-            public_keys:
-              -
-               key:  { get_input: agent_public_key }
-               user: { get_input: agent_user }
+      management_network:
```
Comment (ran-z) [inline types/vcloud-types.yaml]: why was this added as a part of writing the system test?

File: vcloud-blueprint.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ tosca_definitions_version: cloudify_dsl_1_1
 
 imports:
   - http://www.getcloudify.org/spec/cloudify/3.3m6/types.yaml
```
Comment (ran-z) [inline vcloud-blueprint.yaml]: why specific usages of m6 ant not master?

File: vcloud-blueprint.yaml [hunk 3]
```diff
@@ -54,28 +54,118 @@ inputs:
...
-      ################################
+  ###########################################################
+  # A floating ip to be attached to the nodejs host, since
+  # eventually we want to be able to access it
+  # from any machine, on any network.
+  ###########################################################
 
-      - type: node_connected_to_mongo
-        target: mongod
+  nodecellar_floatingip:
```
Comment (EarthmanT) [inline vcloud-blueprint.yaml]: We changed this to "nodecellar_ip". From 3.3m5+ all nodecellar blueprints in this repository should have consistent names for shared node_templates.

File: vcloud-blueprint.yaml [hunk 4]
```diff
@@ -7,7 +7,7 @@ tosca_definitions_version: cloudify_dsl_1_1
```
Comment (EarthmanT) [inline vcloud-blueprint.yaml]: The order of node_templates should be updated to the convention that was established in 3.3m5+. The rules are as follows:

1) First go application nodes
2) Among application nodes, first go front end, then go backend nodes.
3) Then go infrastructure nodes
4) Among, infrastructure nodes, first are compute, then networks, then IPs.
5) Have yet to establish a convention for weird stuff like ssh keys and security groups, but I think it's should be compute, then keys, then groups, but I am open to suggestions.

nodecellar
nodejs:
mongod:
nodejs_host:
mongod_host:
nodejs_security_group:
mongod_security_group:
nodecellar_floatingip:

Since this one is not shared with other nodecellar blueprints, it
